_styles/style.css

```diff
@@ -1,3 +1,13 @@
+.Mealprep {
+    display: block;
+    margin-left:auto;
+    margin-right:auto;
+    height: 350px;
+}
+.Title {
```

Comment:
You may need to change this from previous comment in index.html

Reply:
Why is it that i need to add empty line

Reply:
Good formatting and easier to read especially when you work with bigger code